doc/CONFIGURATION.md

```diff
@@ -81,6 +81,7 @@ See a [sample](./etc/pgmoneta.conf) configuration for running `pgmoneta` on `loc
 | host | | String | Yes | The address of the PostgreSQL instance |
 | port | | Int | Yes | The port of the PostgreSQL instance |
 | user | | String | Yes | The replication user name |
+| extra | | String | No | The source directory for retrieval on the server side |
```

Comment:
You need to describe the requirements for using this - role change, brief setup of the extension and so on.

You also need describe the format of the string with `/path/to/file`, `/path/to/directory/*` and recursive `/path/to/directory/**`